hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/AccessHeatMapEndpoint.java

```diff
@@ -107,4 +107,10 @@ private static void checkIfHeatMapFeatureIsEnabled() {
       throw new WebApplicationException(Response.Status.NOT_FOUND);
     }
   }
+
+  @GET
+  @Path("/healthCheck")
```

Comment:
Can you please add a small java doc explaining the response and purpose of the endpoint.

Reply:
Thanks @ArafatKhan2198 for the review. Added javadoc now for API. Kindly re-review.